modules/ROOT/pages/writing.adoc

```diff
@@ -731,7 +737,7 @@ Given the following example:
       .option("url", SparkConnectorScalaSuiteIT.server.getBoltUrl)
       .option("labels", ":Person:Customer")
       .option("node.keys", "surname")
-      .option("schema.optimization.node.key", true)
+      .option("schema.optimization.node", "KEY")
```

Comment:
```suggestion
      .option("schema.optimization.node.key", "KEY")
```

Reply:
done

Reply:
my bad, it's `.keys` and I don't see the change